spark3-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestMigrateTableProcedure.java

```diff
@@ -99,6 +100,21 @@ public void testMigrateWithOptions() throws IOException {
     sql("DROP TABLE %s", tableName + "_BACKUP_");
   }
 
+  @Test
+  public void testMigrateWithInvalidMetricsConfig() throws IOException {
```

Comment:
I don't think this is related?

Reply:
This test would previously succeed too. It is kind of related to the change to ensure we never fail with a weird exception if the staging step fails. We are fine for now.